docs/team/yuifuku1118.md

```diff
@@ -0,0 +1,33 @@
+layout: page
+title: Yuichiro Fukushima's Project Portfolio Page
+---
+
+### Project: SalesNote
+
+SalesNote is a desktop application that aims to help singapore tailors to manage clients, tasks and sales
+in order to for them to have a more effect order management system and supports a smother daily work which is
+developed by Java and Java FX. 
+
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to save and retrieve order, task, clients from Json file.
+    * What it does: Allows users to save the newly edited data so that user exits the app, the data is saved and able to use it when user opens the app next time.
+    * Justification: This feature was a necessary since we have added a new component of Order and Task.
+    * Highlights: This implementation also enhanced the error check when there was a incorrect manipulation of data json file.
+  
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=yuifuku&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17)
+
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the storage related features (i.e. `exit` and `clear`) [\#139]()
+  
+    * Developer Guide:
+        * Added implementation details of the `storage` component.[\#135]()
+        * Added the all the user stories   [\#35]()
+
+* **Community**:
+    * PRs reviewed (with non-trivial review comments): [\#119](), [\#120](), [\#51](), [\#50]()
```

Comment:
I believe you need to provide links to these pull requests as well so that they would work properly.

Reply:
Oh yes, thanks for commenting! Will fix that right away.